qt/widgets/common/src/MuonFitPropertyBrowser.cpp

```diff
@@ -1248,6 +1178,84 @@ void MuonFitPropertyBrowser::setMultiFittingMode(bool enabled) {
 bool MuonFitPropertyBrowser::isMultiFittingMode() const {
   return m_isMultiFittingMode;
 }
+void MuonFitPropertyBrowser::changeFitFunction(bool enabled) {
```

Comment:
I think this name could be more specific.

Reply:
That is tricky to sum up. Maybe the same as the name of the algorithm `ConvertFitFunctionForMuonTFAsymmetry`?

Reply:
I could do that